lang/en.json

```diff
@@ -16,9 +16,9 @@
         "NoOfActiveBots": "No. of Bots for this site",
         "true": "Yes",
         "false": "No",
-        "Error_empty": " is empty!",
```

Comment:
Just FYI: In case those whitespace were needed to properly display the text in the plugin, I'd recommend you to add them in the code as they were missing in nearly all translations